inline-js/src/Language/JavaScript/Inline.hs

```diff
@@ -0,0 +1,38 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE ExistentialQuantification #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE ViewPatterns #-}
+
+module Language.JavaScript.Inline
+  ( js
+  ) where
+
+import Data.Text as Text
+import qualified Language.Haskell.TH as TH
+import Language.Haskell.TH (Q)
+import Language.Haskell.TH.Quote (QuasiQuoter(..))
+import Language.JavaScript.Inline.Command (eval)
+import Language.JavaScript.Inline.JSCode (codeFromString)
+
+js :: QuasiQuoter
+js =
+  QuasiQuoter
+    { quoteExp = expQQ
+    , quotePat = error "Language.JavaScript.Inline: quotePat"
+    , quoteType = error "Language.JavaScript.Inline: quoteType"
+    , quoteDec = error "Language.JavaScript.Inline: quoteDec"
+    }
+
+expQQ :: String -> Q TH.Exp
```

Comment:
It shall be nice to add a line of comment here for the "actual" type of `TH.Exp`. Typed slices might even be better, but iirc they don't play nicely with source formatters based on `haskell-src-exts`, so just a comment shall be fine I think.